testing/vendor/pretty-format.js

```diff
@@ -0,0 +1,28 @@
+// This file is copied from pretty-format@23.6.0
```

Comment:
Hm - I wonder if we should start a `vendor` directory... If we did it should be at `//third_party/pretty_format.js` 

I haven't been doing this yet, but for these minified blobs that we're never going to edit it seems to be a good organization scheme, so maintainers will know where versions will be bumped.

Yes, I've convinced myself.... please move this to `//third_party/pretty_format.js`

Reply:
@ry Thanks :) Let me confirm should I move diff.ts too?

Reply:
This is a slippery slope though of: "These modules do not have external dependencies"

I can see the advantage of not rewriting everything, but we now have code where we don't have direct type information for in `deno_std`, it is currently written in Flow as part of the [`jest` framework](https://github.com/facebook/jest/tree/master/packages/pretty-format) and the bundle also includes two other dependencies (`ansi-regex` and `ansi-styles` which in turn depend on `color-convert`, `color-name`).  While I can understand the pragmatism of doing this for now, I think we would want to replace it with something that leverage `deno-std/colors` in the long term (which has functional coverage of all the dependencies outside of `pretty-format` that are being dragged in).

I think it is pretty important for code that will essentially be used by every Deno project, including the core itself.

Reply:
Agree. I think so too. However it will take time. I think we have some options.

1. To add this assert to core for now. Then replace external rewrite dependencies as a pure deno module.
2. At first port dependencies to deno(pretty-format and others). Then add this assert to core.

what do you think about it??

Reply:
Agreed.  `colors` actually covers all the other dependencies of `pretty-format` so it is really just porting `pretty-format` which actually would be handy to have "native" in `deno-std` anyways.  It might actually be really good for Deno core as well for any `inspect` functionality, but getting it into `deno_std` would be the first step.  The only challenge is we have avoided adding `colors` to core at the moment.

Reply:
I see. It looks good. @ry How about you? If you are ok, at first I port `pretty-format` to deno then I add this assert.In this case please give me some time.

Reply:
Yes that sounds better. Note that we have a lot of object formatting logic in https://github.com/denoland/deno/blob/master/js/console.ts
Ideally you can reuse some of that code. `deno.inspect` is exposed currently and you can use that - however long term it would probably be better to move all that logic in to `deno_std` since core can reach into std but not the other way around.

Reply:
OK :) I'll fix it.